cypress/e2e/inventory/move-between-instances.cy.js

```diff
@@ -0,0 +1,110 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import permissions from '../../support/dictionary/permissions';
+import TestTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import InventorySearchAndFilter from '../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import Users from '../../support/fragments/users/users';
+import TopMenu from '../../support/fragments/topMenu';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import ItemView from '../../support/fragments/inventory/inventoryItem/itemView';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import InteractorsTools from '../../support/utils/interactorsTools';
+
+
+import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
+
+const successCalloutMessage = '1 holding has been successfully moved.';
+let firstHolding = '';
+let userId;
+let source;
+const item = {
+    instanceName: `Inventory-first-${Number(new Date())}`,
+    ITEM_BARCODE: `123${getRandomPostfix()}`,
```

Comment:
you can use camelcase and lowercase letters, it is not const

Reply:
done